src/App.tsx

```diff
@@ -39,7 +39,7 @@ const App = () => {
               <Link to={"/settings"}>Settings</Link>
             </NavBar>
             <MainContent>
-              <Route path="/library/:id" component={LibraryPage}></Route>
+              <Route path="/library/" component={LibraryPage}></Route>
```

Comment:
More curious than anything, is there a difference between `/library/` and `/library`?

Reply:
I think if I used the `exact` tag they would be different. But this is a typo and just in case I'm wrong I'll fix it.